packages/web-react/src/types/fieldGroup.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  HelperTextProps,
+  SpiritFieldGroupElementPropsWithRef,
+  Validation,
+  ValidationState,
+  ValidationTextType,
+} from './shared';
+import { LabelProps } from './label';
+
+export type FieldGroupElementBaseProps = SpiritFieldGroupElementPropsWithRef;
+
+export interface FieldGroupProps extends FieldGroupElementBaseProps, LabelProps, HelperTextProps, Validation {
+  label: string;
+  isRequired?: boolean;
+  isDisabled?: boolean;
+  isFluid?: boolean;
+  isLabelHidden?: boolean;
+  validationText?: ValidationTextType;
+  validationState?: ValidationState;
```

Comment:
Please, revisit the types. `validationState` is a part of the `Validation` interface.

Reply:
Removed `isRequired`, `validationState` and `UNSAFE_label`.
Added `JSX.Element` to be able add HTML tags